index.js

```diff
@@ -207,28 +198,29 @@ async function printContactenation(files) {
 }
 
 async function getTruffleRoot() {
+  let truffleConfigPath;
   try {
-      const truffleConfiPath = await findUp("truffle.js");
-      return getDirPath(truffleConfiPath);
+    truffleConfigPath = await findUp("truffle.js");
   } catch (error) {
     try {
-      const truffleConfiPath = await findUp("truffle-config.js");
-    }
-    catch (error) {
+      truffleConfigPath = await findUp("truffle-config.js");
```

Comment:
This was a bug. I was using this alternative so getDirPath was never being called

Reply:
Good catch!